components/mpas-albany-landice/cime_config/buildnml

```diff
@@ -237,46 +239,27 @@ def buildnml(case, caseroot, compname):
         lines.append('    <stream name="output"')
         lines.append('            type="output"')
         lines.append('            filename_template="{}.mali.hist.$Y-$M-$D_$S.nc"'.format(casename))
-        lines.append('            filename_interval="0001-00-00_00:00:00"')
+        lines.append('            filename_interval="0000-01-00_00:00:00"')
         lines.append('            clobber_mode="truncate"')
-        lines.append('            output_interval="0000-00-01_00:00:00">')
-        # Note: if output_interval is less than dt, then multiples of that interval will be checked.
-        # There is some performance hit for making this too small.  For now making it 1 day.
+        lines.append('            output_interval="0000-00-05_00:00:00">')
         lines.append('')
         lines.append('    <stream name="basicmesh"/>')
         lines.append('    <var name="xtime"/>')
         lines.append('    <var name="layerCenterSigma"/>')
         lines.append('    <var name="layerInterfaceSigma"/>')
         lines.append('    <var name="thickness"/>')
-        lines.append('    <var name="bedTopography"/>')
         lines.append('    <var name="lowerSurface"/>')
-        lines.append('    <var name="upperSurface"/>')
         lines.append('    <var name="cellMask"/>')
-        lines.append('    <var name="edgeMask"/>')
-        lines.append('    <var name="uReconstructX"/>')
-        lines.append('    <var name="uReconstructY"/>')
         lines.append('    <var name="surfaceSpeed"/>')
-        lines.append('    <var name="basalSpeed"/>')
-        lines.append('    <var name="sfcMassBal"/>')
         lines.append('    <var name="sfcMassBalApplied"/>')
-        lines.append('    <var name="basalMassBalApplied"/>')
         lines.append('    <var name="bareIceAblationApplied"/>')
-        lines.append('    <var name="avgBareIceAblation"/>')
         lines.append('    <var name="avgBareIceAblationApplied"/>')
-        lines.append('    <var name="floatingBasalMassBal"/>')
         lines.append('    <var name="floatingBasalMassBalApplied"/>')
         lines.append('    <var name="groundedBasalMassBalApplied"/>')
-        lines.append('    <var name="calvingThickness"/>')
         lines.append('    <var name="avgCalvingFlux"/>')
         lines.append('    <var name="restoreThickness"/>')
         lines.append('    <var name="dHdt"/>')
-        lines.append('    <var name="ismip6_2dThermalForcing"/>')
-        lines.append('    <var name="ismip6shelfMelt_3dThermalForcing"/>')
-        lines.append('    <var name="orig3dOceanMask"/>')
         lines.append('    <var name="ismip6shelfMelt_TFdraft"/>')
-        lines.append('    <var name="ismip6shelfMelt_zOcean"/>')
-        lines.append('    <var name="ismip6shelfMelt_zBndsOcean"/>')
-        lines.append('    <var name="faceMeltSpeed"/>')
         lines.append('    <var name="faceMeltingThickness"/>')
```

Comment:
It's a little odd to have faceMeltingThickness, but not calvingThickness. Do we intend to eventually create a face-melting flux variable analogous to avgCalvingFlux?

Reply:
Thanks for pointing that out - we do actually have the time-averaged facemelting flux already available.  I've pushed a new commit that replaces faceMeltingThickness with that.